### ISSUE TITLE
Headphones mic not working with WhatsApp video calls? 

### ISSUE THREAD

**Issue by microphonetest** [microphonetest/Headphones-mic-not-working-with-WhatsApp-video-calls-#1]
If you do a lot of voice and video calls on WhatsApp, you might have considered using your headphones or earphones mic for better sound quality. 

However, some users have complained that the headphone mic is not working during WhatsApp video calls in the past few weeks. If you are experiencing this too, you will know that it is such a big mess.

Whatsapp video call headphone mic not working can be caused by a WhatsApp bug to a software and hardware problem with your phone or the headphone microphone you are using. 

Nonetheless, this article contains a series of troubleshooting guides to help you fix earphone mic not working in WhatsApp video calls. This suggests you test the mic and do [mic testing](https://mictests.net).

## Table of contents:

Reboot your device
Disconnect and reconnect the headphones
Adjust the volume of the video call
Ensure the problem is not from the other side
Troubleshoot your headphones
Check for updates
Reinstall WhatsApp
Reset your device
Ok, just reboot your device.

By default, WhatsApp routes the sound to your headphones when connected during a video call. But, the headphone mic not working during WhatsApp video calls sounds weird, and it could be a software or hardware issue. However, you can fix this by rebooting your phone in most cases.

Rebooting your device is much more than just powering off and booting up again. Several software and hardware refreshes take place during the process, which acts as a quick fix for any problems on your phone.

To restart your device, simply press and hold the Power button for about 8-10 seconds. After rebooting your device, try video calling on WhatsApp again to see if the problem is fixed.

## Disconnect and reconnect the headphones.

Next, if the voices of the WhatsApp video calls are coming from your phone's speaker, then it could be a problem with the headset mic itself. A quick way to fix this is to disconnect and reconnect the headphones.

If you are using a wired headphone mic, please unplug it during the video call and reinsert it firmly to fix the problem.
If you connect to a Bluetooth headphone mic, you may need to turn off Bluetooth on your device and the headphones themselves. 

After that, try turning on the headphone mic and connecting them to your phone again.
adjust volume

To make sure your phone isn't muted, try turning up your phone's volume during a WhatsApp video. This is necessary if you are not already hearing sounds from the loudspeaker.

If you can't hear any sound from the video, do yourself a favor and turn up the volume with the Volume Up button on your Android or iPhone. This should fix the problem in most cases. Otherwise, if you hear sounds coming from the loudspeaker, you need to troubleshoot your headphone microphone instead.
 
## Make sure the problem is not from the other side

Although rare, the problem can occur with the person with whom you are making a video call. The person may have inadvertently muted the microphone. To make sure that's not the problem, you can ask the other person to unmute their mic. Do this before moving on to the next step.

## Troubleshoot Your Headphone Microphone

To fix the headphone mic not working during the WhatsApp video call, you need to make sure that the headphone mic is working. This may sound silly to you, but it is part of the process. First, you need to check if the sounds are coming from the headphones/earphones.

An easy way to check this is to play a piece of music on your phone. If it sounds normal, your headphone mic is good. Additionally, you can try making a phone call with a headphone microphone to see if they work. If not, try these troubleshooting guides to fix your Bluetooth or wired headphones.

## Bluetooth headphones? reset it

There are a few ways to fix Bluetooth headphones not working during WhatsApp video calls. But the straightforward way is to reset the Bluetooth device and pair it again with your phone.

**This way:**

On Android, open the Quick Settings panel.
Long press and hold the Bluetooth tile.
Make sure Bluetooth is on.
Under PAIRED DEVICE, you will see your Bluetooth devices.
Tap the gear icon next to your Bluetooth headphones.
On the next screen, tap on Forget Device and confirm it.
Now, make sure that your Bluetooth headphones or earbuds are turned on. You should see it under Available devices on the Bluetooth menu.
Tap on it to pair it again.
Your headphone microphone should be working during WhatsApp video calls by now. Additionally, make sure your headphones have enough battery. Otherwise, it will not work as expected.

## Wired Headphone Mic? try these fixes

On the other hand, if you use wired headphones and it is not producing sound during your WhatsApp video call, try these fixes:

## Unplug the headphones

Check the headphone port on your device. If you find debris inside it, try cleaning it up.
After that, insert the wired headphones into the port to see if it works.
If none of these fixes work for you, you can try the headphone mic one more time to confirm the status of your current headphones/earphones. If they work, you've got your solution.

## Check for updates

Don't ignore it! Another fix for WhatsApp video call headphones not working is to update your phone and WhatsApp itself. There may be some bugs in the current build of the software and WhatsApp running on your phone, which may result in the earphones not working in WhatsApp video calls.

To fix this, please update your WhatsApp from the Play Store on Android or the App Store on iOS. After that, go to your phone's settings to check for system updates and install any available updates.

## How to update an android phone

Launch Settings app
Scroll to System and tap it.
select system update
From there, check for updates online and install all pending updates.
how to update iPhone

open iPhone settings app
Tap General, then select Software Update.
Make sure you are connected to a Wi-Fi network and plug your phone into the charger.
Click on Download and Install.
reinstall Whatsapp
If updating WhatsApp and your phone doesn't solve the problem, the next thing to try is to reinstall WhatsApp. As a result, it will wipe out every corrupted WhatsApp cache and data file that is causing the problem.

However, the downside is that reinstalling WhatsApp will clear your chat history as well. So, make sure to back up your WhatsApp chats to your phone storage before reinstalling WhatsApp. You can do this by going to WhatsApp > Settings > Chats > Chat Backup.

After that, go to Play Store or App Store (on iPhone) to uninstall and reinstall WhatsApp.

## How to reinstall Whatsapp on android

Open Play Store and search
Tap on WhatsApp Messenger.
Below the WhatsApp icon, tap
Wait for a few seconds and tap on Install to reinstall the app.
how to restore Whatsapp on iPhone

From the Home screen or App Library, find WhatsApp and long-press the icon.
Tap Delete and confirm it.
Now, go to App Store and search for WhatsApp.
Click on Install and wait a few moments until WhatsApp is installed on your phone.
After reinstalling WhatsApp on your phone, restore your chats and try to make video calls again to see if it now works through your headphones.

Maybe you should reset your phone...
However, if all the fixes mentioned above are not working for [WhatsApp](https://www.whatsapp.com) video call headphones, then you can try to reset your phone. If your phone is tied to a software issue, or sometimes a hardware issue, then resetting your phone should resolve the issue.

In the meantime, it is essential to back up all important data on your device before performing this action. Resetting your phone will erase all files, apps, and other data stored in the device's memory. After backing up your data, follow these steps to reset your phone:

## How to reset an android phone

open settings
scroll down and select
Tap reset option
Select Erase All Data (Factory Reset)
Finally, tap Erase All Data, confirm your password, and wait for your phone to reboot.
how to reset iPhone

go to setting app
choose general
tap reset
Finally, tap Erase All Content and confirm your password.
After resetting your iPhone or Android, simply restore your data and reinstall WhatsApp. Try making a video call again to see if the problem is fixed.

## Conclusion

Headphones not working during a WhatsApp video call is a rare problem. This does not happen on all devices. However, if you ever experience this on your device, the fixes above will help you out.

The bottom line is to make sure your headphones work naturally in other scenarios. Next, make sure that your phone and WhatsApp are up-to-date to make sure that the problem is not caused by a bug.

If all else fails, you can reset your phone, but it's not always the best solution if you don't have a reliable backup.
